### PR TITLE
feat(clerk-js,types): Organization components localization

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Add localization keys for the org components

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]
👏 👏 👏

File: packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx [hunk 1]
```diff
@@ -57,7 +56,12 @@ export const ActiveMembersList = () => {
       itemCount={organization.membersCount}
       itemsPerPage={ITEMS_PER_PAGE}
       isLoading={!membershipList}
-      headers={['User', 'Joined', 'Role', '']}
+      headers={[
+        localizationKeys('organizationProfile.membersPage.activeMembersTab.headerTitle__user'),
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx:60]: Do you think renaming `headerTitle__user` to `tableHeader__user` makes its intention a little more apparent?
```
        localizationKeys('organizationProfile.membersPage.activeMembersTab.headerTitle__user'),
```

That way its a little harder to confuse with `headerSubtitle__members` that we're using below, which is the actual header subtitle of the page

Comment (nikosdouvlis) [reply packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx:60]: Same applies to the other relevant keys of course

File: packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx [hunk 2]
```diff
@@ -111,7 +115,14 @@ const MemberRow = (props: {
       <Td>
         {isAdmin && (
           <ThreeDotsMenu
-            actions={[{ label: 'Remove member', isDestructive: true, onClick: onRemove, isDisabled: isCurrentUser }]}
+            actions={[
+              {
+                label: localizationKeys('organizationProfile.membersPage.activeMembersTab.action__remove'),
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx:120]: ❓  How about `menuAction__remove`?

File: packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx [hunk 3]
```diff
@@ -111,7 +111,7 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
             direction='col'
             gap={2}
           >
-            <Text>Role</Text>
+            <Text localizationKey={localizationKeys('formFieldLabel__role')} />
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx:114]: ✅

File: packages/clerk-js/src/ui/components/OrganizationProfile/InvitedMembersList.tsx [hunk 4]
```diff
@@ -42,7 +41,12 @@ export const InvitedMembersList = () => {
       itemCount={organization.pendingInvitationsCount}
       itemsPerPage={ITEMS_PER_PAGE}
       isLoading={!invitationList}
-      headers={['User', 'Invited', 'Role', '']}
+      headers={[
+        localizationKeys('organizationProfile.membersPage.activeMembersTab.headerTitle__user'),
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/InvitedMembersList.tsx:45]: ditto

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx [hunk 5]
```diff
@@ -74,9 +70,8 @@ const OrganizationDangerSection = () => {
 
   return (
     <ProfileSection
-      // title={localizationKeys('userProfile.start.profileSection.title')}
       // id='organization-danger'
-      title={'Danger'}
+      title={localizationKeys('organizationProfile.profilePage.dangerSection.title')}
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx:74]: I think we haven't used the word "danger" yet - can you check if it makes sense to replace this one with `destructive`? You can check the `defaultEnglishResource` file for reference

Comment (nikosdouvlis) [reply packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx:74]: I've now realised that "danger" is the name of the section so you can ignore the above unless you think it makes more sense to go that way

File: packages/clerk-js/src/ui/localization/defaultEnglishResource.ts [hunk 6]
```diff
@@ -291,7 +292,7 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
         formHint: 'An email containing a verification code will be sent to this email address.',
         formTitle: 'Verification code',
         formSubtitle: 'Enter the verification code send to {{identifier}}',
-        resendButton: 'Resend link',
+        resendButton: 'Resend code',
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/localization/defaultEnglishResource.ts:295]: Oops.

File: packages/clerk-js/src/ui/localization/defaultEnglishResource.ts [hunk 7]
```diff
@@ -426,16 +427,55 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
...
       successMessage: 'The organization has been updated.',
-      leave: {
-        title: 'Leave organization',
-        messageLine1:
-          'Are you sure you want to leave this organization? You will lose access to this organization and its applications.',
-        messageLine2: 'This action is permanent and irreversible.',
-        successMessage: 'You have left the organization.',
+      dangerSection: {
+        title: 'Danger',
+        leave: {
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/localization/defaultEnglishResource.ts:439]: Nitpicking, but lets rename to `leaveOrganization` to make it a little more explicit

**Review by nikosdouvlis — COMMENTED** [review]
Nicely done

**Pull request merged**